main.py

```diff
@@ -1,12 +1,21 @@
-"""ToDo."""
-from tasks.kyu8 import example
+# -*- coding: utf-8 -*-
+"""Python CodeWars task. 8-kyu
+            https://www.codewars.com/kata/bin-to-decimal
+"""
+from tasks.kyu8 import bin_to_decimal
 
 
-def run_example():
-    """ToDo"""
-    first = input("a: ")
-    second = input("b: ")
-    print(example(first, second))
+def run_bin_to_decimal():
+    """
+    Runs bin_to_decimal function.
+    """
+    number = ''
```

Comment:
Please, delete _number=''_, it's not necessary if you have number = str(input())

Reply:
done